src/components/ComboChart.tsx

```diff
@@ -30,8 +35,18 @@ type IComboChartNonBucketProps = Subtract<IComboChartProps, IComboChartBucketPro
  * is a component with bucket props primaryMeasures, secondaryMeasures, viewBy, filters
  */
 export function ComboChart(props: IComboChartProps): JSX.Element {
-    const primaryMeasures = props.primaryMeasures || [];
-    const secondaryMeasures = props.secondaryMeasures || [];
+    const clonedProps = cloneDeep(props);
+    const { columnMeasures, lineMeasures, viewBy } = clonedProps;
+    const isOldConfig = Boolean(columnMeasures || lineMeasures);
+    const categories = isArray(viewBy) ? [viewBy[0]] : [viewBy];
+
+    if (isOldConfig) {
```

Comment:
can we check this logic by basic shallow enzyme test?

Reply:
Yes, please help to check again